src/scripts/directives.js

```diff
@@ -143,10 +143,15 @@
                 },
                 templateUrl: 'templates/type/radio-input.html',
                 link: function (scope) {
-                    scope.localModel = {
-                        value: scope.model
-                    };
-
+                    scope.localModel = '';
+                    var modelWatchDeregister = scope.$watch('model', function (value) {
+                        if (angular.isDefined(value)) {
+                            scope.localModel = {
+                                value: scope.model
+                            };
+                            modelWatchDeregister();
```

Comment:
ah right I see. Sorry that second watch was chopped off in the PR so I didn't know what this was for. 
Looks good to me then, although perhaps a comment to explain exactly this will help us in future when we ask this same question :)